server/helpers/article.js

```diff
@@ -1,6 +1,7 @@
 
 export const computeOffset = (req) => {
   const page = Object.keys(req.query).length ? parseInt(req.query.page, 10) : 1;
+  // eslint-disable-next-line
```

Comment:
Eslint is throwing an error on isNaN ---> 'no-restricted-globals'. So I disabled the check for this line.